src/OrchardCore.Modules/OrchardCore.Users/Workflows/Activities/RegisterUserTask.cs

```diff
@@ -45,7 +58,7 @@ public bool ConfirmUser
         // The message to display.
         public WorkflowExpression<string> ConfirmationEmailTemplate
         {
-            get => GetProperty(() => new WorkflowExpression<string>());
+            get => GetProperty(() => new WorkflowExpression<string>("Thank you for your interest! Please <a href=\"{{ emailConfirmationUrl }}\">click here</a> to confirm your email."));
```

Comment:
This should be a property of the task itself. Even if the default is exactly this text.

Reply:
removed the default text. Assign it from recipe and make clear in the editor form how to use Workflow.Properties.EmailConfirmationUrl